### PR TITLE
Feature markdown editor

### LINKED ISSUE #624: Kitchen Sink: Build A Markdown Editor
Build a Markdown editor that allows users to write and preview formatted text that you'll add to the kitchen sink. Feel free to use any existing projects of yours or start one from scratch.

For more information about the Kitchen sink; check out:

- https://sink.million.dev/
- https://github.com/aidenybai/million/blob/main/packages/kitchen-sink/README.md 

N.B: **Upon PR review, your usage of Million.js is a plus**

### PR DESCRIPTION
**Pull request by UzairNoman**
**Please describe the changes this PR makes and why it should be merged:**

Added markdown feature.
Resolves #624 

**Status**

- [ ] Code changes have been tested against prettier, or there are no code changes
- [ ] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by UzairNoman** [comment]
@tobySolutions, I created a different branch, used commits from the previous branch, and then created a new PR, but the issue still persists. I couldn't sign into vercel to see what is the error. Is there a way open source contributed can also peek into vercel deployment logs?

**Comment by UzairNoman** [comment]
@tobySolutions, done now :)

**Comment by tobySolutions** [comment]
> @tobySolutions, done now :)

Awesome! Checking out now

File: packages/kitchen-sink/src/style.css [hunk 1]
```diff
@@ -901,6 +901,24 @@ input[type='range'] {
   background-color: #45a049;
 }
 
+.markdown-editor-container {
+  width: 48%;
```
Comment (tobySolutions) [inline packages/kitchen-sink/src/style.css:905]: https://github.com/aidenybai/million/assets/96334363/1741dc30-6823-4cc4-b5fd-41374d2858e4

There is still this issue there @UzairNoman. Can you please review it?

**Comment by UzairNoman** [comment]
Hi @tobySolutions, fixed the responsiveness issues. Let me know if you face any problems still.

**Comment by tobySolutions** [comment]
Okay, cool! Thanks @UzairNoman,

**Pull request merged**